components/x-audio/src/redux/swipe-down-to-minimise.js

```diff
@@ -0,0 +1,31 @@
+const EXPANDED_PLAYER_HEIGHT = 360;
```

Comment:
can we get this from the DOM instead of saving it as a magic number?

Reply:
👍